abr_control/utils/paths.py

```diff
@@ -6,5 +6,6 @@
 if sys.platform.startswith('win'):
     config_dir = os.path.expanduser(os.path.join("~", ".abr_control"))
     cache_dir = os.path.join(config_dir, "cache")
+    database_dir = os.path.joint(cache_dir, "abr_control_db.h5")
 else:
-    cache_dir = os.path.expanduser(os.path.join("~", ".cache", "abr_control"))
+    control_cache = os.path.expanduser(os.path.join("~", "Dropbox", "Data", ".cache", "abr_control"))
```

Comment:
I had to rename this to avoid conflicts with the cache_dir in abr_analyze

Reply:
i've not experienced any conflicts from master branch ...